### DISCUSSION TITLE
Upgrading from v4 to v7

### DISCUSSION THREAD

**Discussion by Leelsmuth** [react-hook-form/react-hook-form#9716] [Q&A]
Hi, I am still using the `"react-hook-form": "^4.10.2" ` version.

I am trying to update to version 7. Because of the numerous breaking changes that would entail, I am trying to see if upgrading incrementally would be a better approach i.e v4 to v5, v5 to v6 and then  v6to v7 but the migration guide for v4 to v5 and v5 to v6 are missing.

https://react-hook-form.com/migration-v6#changes-from-v5-to-v6(returns 404)
https://react-hook-form.com/migration-v5#changes-from-v4-to-v5(returns 404)

Would appreciate if someone could point me to where the guides can be found.

Thanks

**Comment by Moshyfawn** [comment]
Hey, please, don't duplicate your topics ;) There're quite a few discussions already and we're trying to help everyone
https://github.com/react-hook-form/react-hook-form/discussions/4137#discussioncomment-4586128

**Reply by Leelsmuth** [reply]
My apologies, I wasn't sure if I had it in the right place.
